docs/userguide/labels-custom-metadata.md

```diff
@@ -11,34 +11,34 @@ parent = "mn_use_docker"
 # Apply custom metadata
 
 You can apply metadata to your images, containers, or daemons via
-labels. Metadata can serve a wide range of uses. Use labels to add notes or
-licensing information to an image or to identify a host.
+labels. Labels can serve a wide range of uses cases, they can be use to specify
+notes or licensing information to an image or to identify a host.
 
 A label is a `<key>` / `<value>` pair. Docker stores the label values as
-*strings*. You can specify multiple labels but each `<key>` / `<value>` must be
-unique to avoid overwriting. If you specify the same `key` several times but with
-different values, newer labels overwrite previous labels. Docker uses
-the last `key=value` you supply.
+*strings*. You can specify multiple labels but each `<key>` must be
+unique or the value will be overwritten. If you specify the same `key` several
+times but with different values, newer labels overwrite previous labels. Docker
+uses the last `key=value` you supply.
 
 >**Note:** Support for daemon-labels was added in Docker 1.4.1. Labels on
 >containers and images are new in Docker 1.6.0
 
 ## Label keys (namespaces)
 
-Docker puts no hard restrictions on the label `key` you. However, labels with
-simple keys can conflict. For example, you can categorize your images by using a
-chip "architecture" label:
+Docker puts no hard restrictions on the `key` used for a label. However, using
+simple keys can easily leads to conflicts. As an example, let's say you
```

Comment:
I think "As an example, let's say you" is saying the same thing twice, maybe just;

```
For example, you have chosen to categorize your images by CPU architecture
using "architecture" labels in your Dockerfiles: